common/src/main/java/net/william278/husktowns/manager/TownsManager.java

```diff
@@ -656,13 +656,12 @@ public void depositMoney(@NotNull OnlineUser user, @NotNull BigDecimal amount) {
             }
 
             final Town town = member.town();
-            if (!economy.hasMoney(user, amount)) {
+            if(!economy.takeMoney(user, amount, "Town "+town.getId()+":"+town.getName()+" deposit")){
```

Comment:
style: spaces between plus symbols in string concatenation (`+`).
Use IntelliJ ALT+ENTER autoformatting if possible.